### PR TITLE
added comments

### LINKED ISSUE #127: Document exporter and wiper management commands
wiper seems like we'd never want to run it.

### PR DESCRIPTION
**Pull request by jakubkrafka**
closes #127

### CODE REVIEW

File: exporter/management/commands/wiper.py [hunk 1]
```diff
@@ -8,7 +8,12 @@
 
 
 class Command(BaseCommand):
-    help = 'Removes all the exported data'
+    """
+    Starts wiper worker (connects to rabbitmq and consumes messages) responsible for removal
+    of previously exported data. Removes data of particular job (defined in message) only.
```
Comment (jpmckinney) [inline exporter/management/commands/wiper.py:13]: I don't understand. Where is the job to wipe indicated?

Comment (jakubkrafka) [reply exporter/management/commands/wiper.py:13]: This is command to start worker. "Starts wiper worker..." The job is defined in the message "job (defined in message)"

File: exporter/management/commands/wiper.py [hunk 2]
```diff
@@ -8,7 +8,12 @@
...
+
+    It's safe to run multiple workers of this type at the same type.
```
Comment (jpmckinney) [inline exporter/management/commands/wiper.py:15]:
```suggestion
    Starts a worker to delete the files exported from collections in Kingfisher Process.

    It consumes messages from RabbitMQ, which indicate the path components of the directory tree to delete (consisting
    of the spider name and the job ID).

    Multiple workers can run at the same time.
```

**Review by jpmckinney — COMMENTED** [review]
Feel free to merge if suggestions are committed.

File: exporter/management/commands/exporter.py [hunk 3]
```diff
@@ -8,7 +8,13 @@
 
 
 class Command(BaseCommand):
-    help = 'Process, orchestrate and evaluate all jobs and tasks in data registry'
+    """
+    Starts exporter worker (connects to rabbitmq and consumes messages) responsible for export of the
+    data from kingfisher-process db. Zipped data are stored in predefined directory structure and
+    either downloaded directly or sent to flatten tool to be further processed.
+
+    It's safe to run multiple workers of this type at the same type.
```
[the remark this answers is not in the document]
Comment (jakubkrafka) [reply exporter/management/commands/exporter.py:16]: Accepted, just a side note: I'm not a fan of comments, which reveal too much about inner functionality (names of files, directory structure etc.)

**Pull request merged**